spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/notify/microsoft/teams/Section.java

```diff
@@ -0,0 +1,81 @@
+package de.codecentric.boot.admin.notify.microsoft.teams;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Section {
+
+    private String activityTitle;
+    private String activitySubtitle;
+    private String activityImage;
+    private List<Fact> facts = new ArrayList<Fact>();
+    private String text;
+
+    public String getActivityTitle() {
+        return activityTitle;
+    }
+
+    public void setActivityTitle(String activityTitle) {
+        this.activityTitle = activityTitle;
+    }
+
+    public String getActivitySubtitle() {
+        return activitySubtitle;
+    }
+
+    public void setActivitySubtitle(String activitySubtitle) {
+        this.activitySubtitle = activitySubtitle;
+    }
+
+    public String getActivityImage() {
+        return activityImage;
+    }
+
+    public void setActivityImage(String activityImage) {
+        this.activityImage = activityImage;
+    }
+
+    public List<Fact> getFacts() {
+        return facts;
+    }
+
+    public void setFacts(List<Fact> facts) {
+        this.facts = facts;
+    }
+
+    public String getText() {
+        return text;
+    }
+
+    public void setText(String text) {
+        this.text = text;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
Your implementation doesn't rely on the equals/hashcode. For what are these implemented?